app/src/main/java/ch/epfl/skysync/screens/UserDetails.kt

```diff
@@ -35,10 +55,43 @@ fun UserDetailsScreen(navController: NavHostController, flightsViewModel: Flight
         bottomBar = { AdminBottomBar(navController = navController) },
         floatingActionButtonPosition = FabPosition.End,
     ) { paddingValues ->
-      FlightsList(userFlights.value, lightOrange, paddingValues, "Completed Flights") {
-          selectedFlight ->
+      FlightsList(userFlights.value, paddingValues) { selectedFlight ->
         navController.navigate(Route.ADMIN_FLIGHT_DETAILS + "/$selectedFlight")
       }
     }
   }
 }
+
+@Composable

```

Comment:
Why did you redefine `FlightList` here? Because there is already a component `FlightList` that does the same thing I think

Reply:
Because FlightList comes with a colored header and i think it look weird here so i redefined it to just have the flights

Reply:
now i create function in FlightList called FlightListContent that only has the list and i call iton UserDetails instead of redefining FlightList